test/htmlwriter.jl

```diff
@@ -104,6 +111,8 @@ end
 
     mktempdir() do tmpdir
         versionfile = joinpath(tmpdir, "versions.js")
+        redirectfile = joinpath(tmpdir, "index.html")
+        devurl = "dev"
```

Comment:
Would be good with a test for for the case where no versions exist (to verify that `dev` is picked).

Reply:
I've added the tests in case6. (see comments)